backend/test/test_qa.py

```diff
@@ -257,6 +257,53 @@ def test_qa_replay(
     assert data["resources"][0]["path"]
 
 
+def test_qa_stats(
+    crawler_crawl_id,
+    crawler_auth_headers,
+    default_org_id,
+    qa_run_id,
+    qa_run_pages_ready,
+):
+    # We'll want to improve this test by having more pages to test
+    # if we can figure out stable page scores to test against
+    r = requests.get(
+        f"{API_PREFIX}/orgs/{default_org_id}/crawls/{crawler_crawl_id}/qa/{qa_run_id}/stats?screenshotThresholds=0.7,0.9&textThresholds=0.7,0.9",
+        headers=crawler_auth_headers,
+    )
+    assert r.status_code == 200
+
+    data = r.json()
+    assert data["screenshotMatch"] == [{"lowerBoundary": "0.9", "count": 1}]
```

Comment:
Should this include 0.0 and 0.7 as well?

Reply:
Since there's no documents in the 0-0.7 and 0.7-0.9 ranges, `$bucket` doesn't include them in the output. We could manually add them to be more explicit?

Reply:
Modified to include all thresholds in result even if count is 0